packages/signers/package.json

```diff
@@ -63,6 +65,8 @@
     "viem": "^1.16.2"
   },
   "peerDependencies": {
+    "@web3auth/base": "^7.1.0",
+    "@web3auth/modal": "^7.1.1",
     "magic-sdk": "^21.2.0"
```

Comment:
this will make our signers have multiple peer Dependencies that are actually not peer dependencies. Would there be any better way to manage this? Devs with this flow will have to ignore the other peer deps that are not relevant for them.

wdyt? @moldy530 @avasisht23

Reply:
could you say more? why would signers need multiple peer dependencies?

It's up to the developer to install in their project the extra dependency needed for the specific signer they choose. and if they swap out, they would need to install/uninstall as necessary.

Reply:
yea I think this is fine, they would only need to install the peer deps they want. It's possible we can get away with having these as deps instead and then leave it to developer's tree-shaking set up to drop the packages they don't need, but that comes at the cost of increased `npm install` times in dev

Reply:
not: making these optionalDependencies so it's not required to install. Think I might have conflated peer with optional and led to this confusion.